tests/chkstk-test.c

```diff
@@ -2,12 +2,17 @@
    Tests stack probing
 */
 
-void f1(char* data) 
+#define BIG_STACK_ALLOC_SIZE 300000
+
+void write_to_big_stack_alloc(char* stack_data) 
 {
-    *data = 0;
+    stack_data[0] = 18;
+    stack_data[BIG_STACK_ALLOC_SIZE - 1] = 19;

```

Comment:
What is a reason to change the test which was reproducing the crash?

Reply:
I would like to test the top and the bottom of the stack bounds.
We could not do this previously because of stack probing not being implimented.

Reply:
top of stack was tested by *data = 0
bottom check is not needed for this case
stack_data[BIG_STACK_ALLOC_SIZE - 1] = 19;

Reply:
This PR is getting large. Will follow-up in next PR to avoid merging issues.